src/Admin/App.js

```diff
@@ -73,11 +63,6 @@ class App extends React.Component {
     userDetailsDispatch({ ...user.data })
   }
 
-  hideTermsAndConditions = () => {
```

Comment:
not sure what this code was doing in here, perhaps from when terms opened a popup :confused: